IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -183,7 +185,7 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(__unused NSEr
 - (NSDictionary *)jsonDictionary
 {
     MSID_TRACE;
-    NSMutableDictionary *dictionary = [NSMutableDictionary dictionary];
+    NSMutableDictionary *dictionary = [NSMutableDictionary dictionaryWithDictionary:_additionalInfo];
```

Comment:
Shouldn't this merging be handled by _json variable already? Additional info is necessary if we write some extra info into there during the runtime, and is partially "legacy" thing as ADAL used to store all extra fields in there.